commands/worker/main.go

```diff
@@ -24,14 +27,15 @@ import (
 )
 
 // Version of the runner.
-const Version = "0.0.1"
+const Version = "0.0.2"
```

Comment:
```suggestion
const Version = "0.1.1"
```
Follow semantic versioning. This is quite a drastic change in functionality.

https://en.wikipedia.org/wiki/Software_versioning You can read more here, if not already aware.

Reply:
My older comment was incorrect. I wasn't thinking it through. Should have always been bumped to 0.1.1

Reply:
Hmm, got it. 
0.1.1 because: it's a minor version change: as it is backwards compatible. and a patch version as well is applied. We won't call this a major version change, because the code changes aren't breaking compatibility...